src/modules/AuthModule/components/ChangePass/ChangePass.tsx

```diff
@@ -1,8 +1,211 @@
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import { IconButton, TextField, ThemeProvider, createTheme } from "@mui/material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { useFormik } from "formik";
+import * as React from "react";
+import { toast } from "react-toastify";
+import { axiosInstanceWithHeaders } from "../../../../axiosConfig/axiosInstance";
+import { FormChangePass } from "../../../../interfaces/Auth";
+import { useNavigate } from "react-router-dom";
+import { validationSchema } from "../../../../validations/validations";
+
+
+const defaultTheme = createTheme();
 
 const ChangePass = () => {
+  const [showOldPassword, setShowOldPassword] = React.useState(false);
+  const [showNewPassword, setShowNewPassword] = React.useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = React.useState(false);
+  const navigate = useNavigate()
+
+  const handleClickShowOldPassword = () => setShowOldPassword((show) => !show);
+  const handleMouseDownOldPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const handleClickShowNewPassword = () => setShowNewPassword((show) => !show);
+  const handleMouseDownNewPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const handleClickShowConfirmPassword = () =>
+    setShowConfirmPassword((show) => !show);
+  const handleMouseDownConfirmPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const style = {
+    position: "absolute" as "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    bgcolor: "background.paper",
+    boxShadow: 24,
+    borderRadius: 8,
+    p: 4,
+  };
+  const initialValues: FormChangePass = {
+    oldPassword: "",
+    newPassword: "",
+    confirmPassword: "",
+   
+  };
+
+  async function onSubmit(values: FormChangePass) {
+    try {
+      let response = await axiosInstanceWithHeaders.post(
+        `admin/users/change-password`,
+        values
+      );
+      toast.success('Your Password has changed successfully' || response?.data?.message)
+      navigate('/login')
+    } catch (error:any) {
+      toast.error("there's an eror" || error?.response?.data?.message)
+
+    }
+  }
+
+  const formik = useFormik({
+    initialValues,
+    onSubmit,
+    validationSchema
+  });
+
+
   return (
-    <div>ChangePass</div>
-  )
-}
+    <ThemeProvider theme={defaultTheme}>
+   
+          <Box sx={style}>
+          <h1 className="p-5 text-4xl font-serif font-bold text-center" >
+            <span className="text-[#152C5B] font-[4rem]" >Stay</span>cation.
+          </h1>
+            <Typography
+              id="transition-modal-title"
+              style={{color:'#152C5B' , fontSize:'1.4rem'}}
```

Comment:
Using hardcoded colors like this will make us need to handle light and dark colors for them manually. On the other hand, if we stick to the MUI colors, the theme colors will change automatically without needing to exert any effort in managing the colors.